app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    if command.count(" ") > 2 or command.count(" ") < 2:
+        return "wrong command"
```

Comment:
```suggestion
    if command.count(" ") != 2
        return "wrong command"
```


but your check  doesn't work when I type `сp cp cp`

Reply:
Just check as:
`if len(command.split()) != 3`